tests/integration/targets/lookup_passwordstore/tasks/tests.yml

```
@@ -70,150 +80,22 @@
-
-- name: Verify password
-  assert:
-    that:
-      - readpass == newpass
-
-- name: Fetch password from non-existing file using missing=empty
-  set_fact:
-    readpass: "{{ query('community.general.passwordstore', 'test-missing-pass missing=empty') }}"
-
-- name: Verify password
-  assert:
-    that:
-      - readpass == [ none ]
-
-# As inserting multiline passwords on the commandline would require something
-# like expect, simply create it by using default gpg on a file with the correct
-# structure.
-- name: Create the YAML password content
-  copy:
-    dest: "~/.password-store/test-yaml-pass"
-    content: |
-      testpassword
-      key: |
-        multi
-        line
-
-- name: Read .gpg-id from .password-store
-  set_fact:
-    gpgid: "{{ lookup('file', '~/.password-store/.gpg-id') }}"
-
-- name: Encrypt the file using the gpg key
-  command: "{{ gpg2_bin }} --batch --encrypt -r {{ gpgid }} ~/.password-store/test-yaml-pass"
-
-- name: Fetch a password with YAML subkey
-  set_fact:
-    readyamlpass: "{{ lookup('community.general.passwordstore', 'test-yaml-pass subkey=key') }}"
-
-- name: Read a yaml subkey
-  assert:
-    that:
-      - readyamlpass == 'multi\nline'
-
-- name: Create a non-YAML multiline file
-  copy:
-    dest: "~/.password-store/test-multiline-pass"
-    content: |
-      testpassword
-      random additional line
-
-- name: Read .gpg-id from .password-store
-  set_fact:
-    gpgid: "{{ lookup('file', '~/.password-store/.gpg-id') }}"
-
-- name: Encrypt the file using the gpg key
-  command: "{{ gpg2_bin }} --batch --encrypt -r {{ gpgid }} ~/.password-store/test-multiline-pass"
-
-- name: Fetch password from multiline file
-  set_fact:
-    readyamlpass: "{{ lookup('community.general.passwordstore', 'test-multiline-pass') }}"
-
-- name: Multiline pass only returns first line
-  assert:
-    that:
-      - readyamlpass == 'testpassword'
-
-- name: Fetch all from multiline file
-  set_fact:
-    readyamlpass: "{{ lookup('community.general.passwordstore', 'test-multiline-pass returnall=yes') }}"
-
-- name: Multiline pass returnall returns everything in the file
-  assert:
-    that:
-      - readyamlpass == 'testpassword\nrandom additional line'
-
-- name: Create a password in a folder
-  set_fact:
-    newpass: "{{ lookup('community.general.passwordstore', 'folder/test-pass length=8 create=yes') }}"
-
-- name: Fetch password from folder
-  set_fact:
-    readpass: "{{ lookup('community.general.passwordstore', 'folder/test-pass') }}"
-
-- name: Verify password from folder
-  assert:
-    that:
-      - readpass == newpass
-
-- name: Try to read folder as passname
-  set_fact:
-    newpass: "{{ lookup('community.general.passwordstore', 'folder') }}"
-  ignore_errors: true
-  register: eval_error
+- name: Initialise gopass passwordstore
+  command: gopass init --path {{ lookup('env','HOME') }}/.gopass-store ansible-test
```

This should probably work:
```suggestion
  command: gopass init --path $HOME/.gopass-store ansible-test
```